hooks/tk-multi-publish2/basic/upload_version.py

```diff
@@ -247,10 +247,12 @@ def publish(self, settings, item):
 
         if settings["Upload"].value:
             self.logger.info("Uploading content...")
+            # the upload function triggers calls to os.path.* methods that may not give expected results when incput string is encoded in utf-8,
```

Comment:
typo: `incput`

suggest: limit the length of comment lines to `80` characters for consistency with the rest of this code.